action.yaml

```diff
@@ -13,31 +13,39 @@ inputs:
   python_version:
     description: version of python to use
     required: false
-    default: 3.8
+    default: 3.10
 
-  aws_access_key_id:
-    description: access key id of the ci user
-    required: true
-
-  aws_secret_access_key:
-    description: secret access key of the ci user
-    required: true
+  aws_region:
+    description: name of AWS region
+    required: false
+    default: "us-east-1"
 
   role_to_assume:
     description: iam role assumed for deployments
     required: true
 
+  role_session_name:
+    description: name for iam role session
+    required: false
+    default: "GitHubActions"
+
+  role_duration_seconds:
+    description: >
+      credentials session duration when assuming role
+    required: false
+    default: 1200
+
 runs:
 
   using: composite
 
   steps:
 
       - name: Checkout repository
```

Comment:
<img width="874" alt="image" src="https://user-images.githubusercontent.com/10556275/225077846-cfa0055a-c05d-4454-932f-b1063d45a872.png">

Lets makes sure we add these permissions.

Reply:
Oh... maybe we don't need to add this here?

Reply:
I didn't add them initially because I thought they were specific to the job that actually integrates in this action?

Reply:
Ya I'm not sure you can add them here? I think we just need to add in the instructions for "how to use this action"